### PR TITLE
ENG-129: Specify mongoose version to 6

### PR DESCRIPTION
**Pull request by kkyriakidis**
https://workable.atlassian.net/browse/ENG-129

### CODE REVIEW

**Comment by nikostoulas** [comment]
@kkyriakidis Should we first make a release with mongoose 5.* to support current clients?

**Comment by kkyriakidis** [comment]
> 

I created a different PR for this
https://github.com/Workable/orka/pull/285

File: examples/mongodb-example/config.js [hunk 1]
```diff
@@ -4,12 +4,5 @@ module.exports = {
 
   mongodb: {
     url: 'mongodb://localhost/orka',
-    // mongoose options for tuning mongodb.
-    // Details: https://mongoosejs.com/docs/connections.html
-    options: {
```
Comment (nikostoulas) [inline examples/mongodb-example/config.js:9]: ❓ Are there no options available with mongoose 6?

**Review by nikostoulas — COMMENTED** [review]
LGTM

File: examples/mongodb-example/config.js (hunk 1 above)
Comment (nikostoulas) [reply examples/mongodb-example/config.js:9]: What about this @kkyriakidis ? Is there no options now?

File: docs/configuration.md [hunk 2]
```diff
@@ -227,10 +227,6 @@ You can find the default values below:
     "url": "",
     "options": {
       // options that go into mongoose connect
-      "useNewUrlParser": true,
```
Comment (nikostoulas) [inline docs/configuration.md:230]: Maybe add a link to mongoose as a comment?

**Review by nikostoulas — APPROVED** [review]

**Pull request merged**